cmd/create/cluster/cmd.go

```diff
@@ -644,7 +653,16 @@ func init() {
 		"Maximum amount of memory, in GiB, in the cluster.",
 	)
 
-	// TODO: handle GPU limitations
+	flags.StringVar(
+		&args.autoscalerResourceLimits.GPULimitsInput,
+		"gpu-limits",
+		"",
+		"The stringified GPULimits array. An array of GPULimtis. Each GPULimit is an array comprised of 3 valuse: [gpuType, minGPU, maxGPU]."+
+			"gpuType: The type of GPU node to deploy. Options: [nvidia.com/gpu, amd.com/gpu]."+
```

Comment:
there could be other options I think, so let's omit those from the description
```suggestion
			"gpuType: The type of GPU node to deploy."+
```

Reply:
or maybe we can change "options" to "examples". something like:
```
"gpuType: The type of GPU node to deploy, e.g.: nvidia.com/gpu, amd.com/gpu, etc."+
```

Reply:
Thanks for the clarification, I will go with the example strategy.